releasenotes/notes/remove-deprecated-ops-d01b83362c3557ca.yaml

```
@@ -0,0 +1,5 @@
+---
+deprecations:
+  - |
+    Remove :func:`~qiskit.circuit.measure.measure` and :func:`~qiskit.circuit.reset.reset` 
```

Thanks for adding the reno!

Since the functions will be removed I think we cannot link to them anymore 🙂 Also we usually put this under the `upgrade` part since it's an API change -- the deprecation technically already happened before 😄 
```suggestion
upgrade:
  - |
    Remove ``qiskit.circuit.measure.measure`` and ``qiskit.circuit.reset.reset`` 
```